packages/evolution-backend/src/services/audits/auditChecks/checks/__tests__/trip/testHelper.ts

```
@@ -0,0 +1,38 @@
+import { Household } from 'evolution-common/lib/services/baseObjects/Household';
+import { Home } from 'evolution-common/lib/services/baseObjects/Home';
+import { Interview } from 'evolution-common/lib/services/baseObjects/interview/Interview';
+import { TripAuditCheckContext } from '../../../AuditCheckContexts';
+
+export const createMockTrip = (overrides: Partial<Trip> = {}, validUuid = uuidV4()) => {
+    return {
+        _uuid: validUuid,
+        startDate: '2023-10-15',
+        ...overrides
+    } as Trip;
+};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)